<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LSQDialog</class>
 <widget class="QDialog" name="LSQDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Least Squares Refinement</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="refinementRequestGroup">
         <property name="title">
          <string>Refinement Request</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="diagonalRadio">
            <property name="text">
             <string>Diagonal</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="fullMatrixRadio">
            <property name="text">
             <string>Full Matrix</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sfcOnlyRadio">
            <property name="text">
             <string>S.F.C. only</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="refinementConditionsGroup">
         <property name="title">
          <string>Refinement Conditions</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="dampingFactorLabel">
            <property name="text">
             <string>Damping Factor</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="dampingFactorSpin">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="reflectionsCutoffLabel">
            <property name="text">
             <string>Reflections cutoff</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="reflectionsCutoffSpin">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>9</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="cyclesLabel">
            <property name="text">
             <string># of Cycles</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="cyclesSpin">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="weightingSchemeGroup">
         <property name="title">
          <string>Weighting Scheme</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QComboBox" name="weightingSchemeCombo"/>
          </item>
          <item>
           <widget class="QPushButton" name="modifyWeightButton">
            <property name="text">
             <string>Modify weight parameters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="refineWeightCheck">
            <property name="text">
             <string>Refine weight parameters</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="modifyAtomsTab">
      <attribute name="title">
       <string>Modify Atoms Parameters</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="observationsParametersGroup">
     <property name="title">
      <string>Observations &amp; Parameters</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="observationsLabel">
        <property name="text">
         <string>Observations: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="parametersLabel">
        <property name="text">
         <string>Parameters: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ratioLabel">
        <property name="text">
         <string>Ratio: 0.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LSQDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
